jc/parsers/find.py

```diff
@@ -0,0 +1,121 @@
+"""jc - JSON Convert `find` command output parser
+
+Usage (cli):
+
+    $ find | jc --find
+
+Usage (module):
+
+    import jc
+    result = jc.parse('find', find_command_output)
+
+Schema:
+
+    [
+      {
+        "directory path":           string,
+        "file or empty directory:   string
+      }
+    ]
```

Comment:
Can we shorten the key names and change spaces to underscore? Something like: `path` and `file`. Or maybe something like `node` makes more sense instead of `file` since it can represent an empty directory?